code-coverage-report/build.gradle.kts

```diff
@@ -6,21 +6,5 @@ plugins {
 }
 
 dependencies {
-  jacocoAggregation(projects.cast.java)
-  jacocoAggregation(projects.cast.java.ecj)
-  jacocoAggregation(projects.cast.js)
-  jacocoAggregation(projects.cast.js.html.nuValidator)
-  jacocoAggregation(projects.cast.js.nodejs)
-  jacocoAggregation(projects.cast.js.rhino)
-  jacocoAggregation(projects.core)
-  jacocoAggregation(projects.dalvik)
-  jacocoAggregation(projects.ide)
-  jacocoAggregation(projects.ide.jdt)
-  jacocoAggregation(projects.ide.jdt.test)
-  jacocoAggregation(projects.ide.jsdt)
-  jacocoAggregation(projects.ide.jsdt.tests)
-  jacocoAggregation(projects.ide.tests)
-  jacocoAggregation(projects.scandroid)
-  jacocoAggregation(projects.shrike)
-  jacocoAggregation(projects.util)
+  rootProject.subprojects { plugins.withId("jacoco") { jacocoAggregation(this@subprojects) } }
```

Comment:
Coincidentally I think I got the Codecov integration working earlier today!  Its results look ok, but did you happen to check if `subprojects` covers projects nested at arbitrary depths, like `cast/js/rhino`?

Reply:
> Coincidentally I think I got the Codecov integration working earlier today!

W00t! 👏

> did you happen to check if `subprojects` covers projects nested at arbitrary depths, like `cast/js/rhino`?

Not specifically. However, I generated the aggregated coverage report before and after this change, and found them to be identical.

Reply:
Hmm, let's just place it safe: I'll replace `rootProject.subprojects` with `rootProject.allprojects`. Then there should be no doubt as to whether we got everything.